### ISSUE TITLE
cannot install customizable shadcn component

### ISSUE THREAD

**Issue by hiroaki2020** [Yonom/assistant-ui#1178]
Following [this guide](https://www.assistant-ui.com/docs/ui/shadcn-ui/AssistantModal), I tried to add customizable shadcn component but failed. My cli log is below.

```
my-cli % npx shadcn@latest init
✔ Preflight checks.
✔ Verifying framework. Found Next.js.
✔ Validating Tailwind CSS.
✔ Validating import alias.
✔ Which style would you like to use? › New York
✔ Which color would you like to use as the base color? › Neutral
✔ Would you like to use CSS variables for theming? … no / yes
✔ Writing components.json.
✔ Checking registry.
✔ Updating tailwind.config.ts
✔ Updating src/styles/globals.css
✔ Installing dependencies.
✔ Created 1 file:
  - src/lib/utils.ts

Success! Project initialization completed.
You may now add components.

my-cli % npx assistant-ui@latest add modal
Invalid configuration found in /path/to/components.json.
other-package-script process exited with code 1
```

**Comment by Yonom** [comment]
Thank you, I am working on a fix...

**Comment by Yonom** [comment]
@hiroaki2020 this should be fixed now with the new rewritten CLI! let me know if you're still experiencing issues

**Comment by hiroaki2020** [comment]
> @hiroaki2020 this should be fixed now with the new rewritten CLI! let me know if you're still experiencing issues

I tried again and got another error.
```
my-cli % npx assistant-ui@latest add modal
Need to install the following packages:
assistant-ui@0.0.21
Ok to proceed? (y) y
⠇ Checking registry.

Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

The component at https://r.assistant-ui.com/shadcn/modal was not found.
It may not exist at the registry. Please make sure it is a valid component.

other-package-script process exited with code 1
```

**Comment by haaln** [comment]
> I tried again and got another error.

try `npx assistant-ui@latest add assistant-modal` (i.e. component is called assistant-modal and not modal)

**Comment by Yonom** [comment]
Thanks for flagging this, the name should actually be `modal`!

fixed in #1198
